tests/requirements/base.txt

```diff
@@ -10,3 +10,5 @@ pytz > dev
 selenium
 sqlparse
 tblib
+asgiref
+daphne
```

Comment:
If you mean ``setup.py``, then yes, let's not duplicate them. I had to get this in there to get the PR test runner working.